wbia/control/manual_annot_funcs.py

```diff
@@ -1973,7 +1973,7 @@ def get_annot_yaws(ibs, aid_list, assume_unique=False):
         tau = 2 * pi
 
     SeeAlso:
-        wbia.const.VIEWTEXT_TO_YAW_RADIANS
+        const.VIEWTEXT_TO_YAW_RADIANS
```

Comment:
We should replace all of these instances with `wbia.constants.` since it is documentation